HW9/task2.py

```diff
@@ -0,0 +1,10 @@
+# Створити програму, яка буде приймати число і друкувати відповідне число в послідовності Фібоначчі,
+# використовуючи рекурсію.
+while True:
+    def fibonacci(n):
+        if n in [0, 1]:
+            return n
+        return fibonacci(n-1)+fibonacci(n-2)
```

Comment:
nit: better to add spaces
```suggestion
        return fibonacci(n - 1) + fibonacci(n - 2)
```